pinot-query-planner/src/test/resources/queries/PhysicalOptimizerPlans.json

```diff
@@ -604,6 +605,44 @@
       }
     ]
   },
+  "physical_opt_run_in_broker": {
+    "queries": [
+      {
+        "description": "(run-in-broker) Pagination on group-by results",
+        "sql": "SET usePhysicalOptimizer=true; SET useLiteMode=true; SET runInBroker=true; EXPLAIN PLAN FOR WITH tmp AS (SELECT col1, col2, col3, COUNT(*) FROM a WHERE col1 = 'foo' GROUP BY col1, col2, col3 ORDER BY col2) SELECT * FROM tmp LIMIT 100,400",
+        "output": [
+          "Execution Plan",
+          "\nPhysicalSort(offset=[100], fetch=[400])",
```

Comment:
Note that unlike other plans, the top-most plan node here is not an exchange. Reason is that the exchange in the sub-tree sends data to the broker directly, and there's no exchange required after that.

If there's a collation trait to be met, that'll be met by adding a sort.